src/lib/permissions/pages/policies/interfaces.ts

```diff
@@ -0,0 +1,9 @@
+import type { Bounty, Page } from '@prisma/client';
+
+import type { PermissionFilteringPolicyFnInput } from '../../core/policies';
+import type { PagePermissionFlags } from '../interfaces';
+
+export type PageResource = Pick<Page, 'id' | 'proposalId' | 'convertedProposalId' | 'type' | 'createdBy'> & {
+  bounty: Pick<Bounty, 'createdBy' | 'spaceId' | 'status'> | null;
```

Comment:
Not sure why PageResource is used for, but it seems like Bounty should be handled by a bounty permissions policy. Is this temporary?

Reply:
Yeah there’s probably a way to simplify this. I’ll make a note to handle it m